drivers/syscon/Kconfig

```diff
@@ -0,0 +1,38 @@
+# SYSCON configuration options
+
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# SYSCON options
+#
+menuconfig SYSCON
+	bool "SYSCON (System Controller) drivers"
+	help
+	  SYSCON (System Controller) drivers
```

Comment:
Please add a bit more about SYSCON in the help here about when SYSCON should be utilized.

Reply:
ok